tests/filecheck/func_ops.xdsl

```diff
@@ -15,21 +15,21 @@ builtin.module() {
     func.return()
   }
 
-  // CHECK: func.func() ["sym_name" = "call_void", 
+  // CHECK: func.func() ["sym_name" = "call_void",
   // CHECK-NEXT:  func.call() ["callee" = @call_void]
   // CHECK-NEXT:  func.return()
   // CHECK-NEXT:}
 
-  func.func() ["sym_name" = "arg_rec", "function_type" = !fun<[!i32], [!i32]>, "sym_visibility" = "private"] {
-  ^0(%0 : !i32):
-    %1 : !i32 = func.call(%0 : !i32) ["callee" = @arg_rec]
-    func.return(%1 : !i32)
+  func.func() ["sym_name" = "arg_rec", "function_type" = !fun<[!test.type<"int">], [!test.type<"int">]>, "sym_visibility" = "private"] {
```

Comment:
Assuming I am reading this filecheck test, why should I need to go to test dialect to understand what is going on?

Reply:
You mean when a new user wants to understand how IRs look like or how the func dialect looks like?

Reply:
@georgebisbas gentle ping

Reply:
I was thinking of how understandable is a test from someone that has not seen the test dialect

Reply:
It isn't and I don't think it's supposed to be. IMO, people should start with the documentation. If there is no docu, they will go look at test cases (at least that's how I am doing it). So, I think, we should not design our testing with new user readability in mind.